src/scribe_data/resources/language_meta_data.json

```diff
@@ -15,56 +16,64 @@
       "iso": "en",
       "qid": "Q1860",
       "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": []
+      "ignore-words": [],
+      "word-types": ["nouns", "verbs", "translated_words"]
     },
     {
       "language": "french",
       "iso": "fr",
       "qid": "Q150",
       "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": ["XXe"]
+      "ignore-words": ["XXe"],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
     },
     {
       "language": "german",
       "iso": "de",
       "qid": "Q188",
       "remove-words": ["of", "the", "The", "and", "NeinJa", "et", "redirect"],
-      "ignore-words": ["Gemeinde", "Familienname"]
+      "ignore-words": ["Gemeinde", "Familienname"],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
     },
     {
       "language": "italian",
       "iso": "it",
       "qid": "Q652",
       "remove-words": ["of", "the", "The", "and", "text", "from"],
-      "ignore-words": ["The", "ATP"]
+      "ignore-words": ["The", "ATP"],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
```

Comment:
I don't think that we have the prepositions set up for all of the languages :)

Reply:
Note that the above was a main justification for moving away from this, as it's 100% going to be a thing that will happen again and again as Scribe-Data develops and new word types are added to new languages. Best to just check what's actually in the `language_data_extraction` directories.